pkg/query/config.go

```diff
@@ -39,3 +44,29 @@ func LoadConfigs(confYAML []byte) ([]Config, error) {
 	}
 	return queryCfg, nil
 }
+
+// BuildQueryConfig returns a query client configuration from a static address.
+func BuildQueryConfig(queryAddrs []string) ([]Config, error) {
+	configs := make([]Config, 0, len(queryAddrs))
+	for _, addr := range queryAddrs {
+		if addr == "" {
+			return nil, errors.New("static querier address cannot be empty")
+		}
+		// If addr is missing schema, add http.
+		if !strings.Contains(addr, "://") {
+			addr = fmt.Sprintf("http://%s", addr)
+		}
+		u, err := url.Parse(addr)
```

Comment:
we could also verify that the scheme is `http` or `https`? It would catch typos like `ttp://example.com` which is a valid URL. 

@bwplotka WDYT?

Reply:
I have added support for it.

Reply:
Thanks :+1: